common/remote/rmtsmtp.cpp

```diff
@@ -574,6 +592,8 @@ class CMailInfo
     {
         header.append(senderHeader).append(sender.get()).append("\r\n");
         header.append(toHeader).append(to.str()).append("\r\n");
+        if (!cc.isEmpty())
+            header.append(ccHeader).append(cc.str()).append("\r\n");
         header.append(subjectHeader).append(subject.get()).append("\r\n");
```

Comment:
Forgot to add the bcc entries.

Reply:
I see richard's comment... and your answer!

Reply:
I wonder if this is an indication that I should add a comment describing why bcc is omitted, so a future maintainer doesn't "fix" it.

Reply:
probably worthwhile, although it was because two items were added at the same time that it stood out.